### ISSUE TITLE
Create Words from a set of ... "words"

### ISSUE THREAD

**Issue by simpsus** [danbernier/WordCram#25]
Usecase: The cram shall be filled with the same few words, but in different sizes. The normal Wordcram methods (fromText-likes) will eliminate the duplicate words and the resulting cram will show each word only once. 

So it would be nice to have a helper class which yu can pass a word and how often it shall appear with what weights and this class build the Word[] for you.

The applications go mostly towards ShapeBased crams where you want to fill a shape with a distinct set of words (the age or silhouette with the name of the person or such things).
